coffea/jetmet_tools/JetTransformer.py

```diff
@@ -29,11 +25,11 @@ def _update_jet_ptm(corr, jet, fromRaw=False):
     as we apply corrections and smearings.
     """
     if fromRaw:
-        jet._content._contents['__fast_pt'] = corr * jet.ptRaw.content
-        jet._content._contents['__fast_mass'] = corr * jet.massRaw.content
+        jet._content._contents["__fast_pt"] = corr * jet.ptRaw.content
+        jet._content._contents["__fast_mass"] = corr * jet.massRaw.content
     else:
-        jet._content._contents['__fast_pt'] = corr * jet.pt.content
-        jet._content._contents['__fast_mass'] = corr * jet.mass.content
+        jet._content._contents["__fast_pt"] = corr * jet.pt.content
+        jet._content._contents["__fast_mass"] = corr * jet.mass.content
 
 
 # the class below does use hacks of JaggedCandidateArray to achieve the desired behavior
```

Comment:
Is this file still in use now that JCA and ak0 are discontinued?

Reply:
deleted